src/syrupy/extensions/amber.py

```diff
@@ -28,7 +29,7 @@ class DataSerializer:
 
     class MarkerDepthMax:
         def __repr__(self) -> str:
-            return "..."
+            return SYMBOL_ELLIPSIS
```

Comment:
Even though we use the same symbol for context & max depth, semantically they're different and could theoretically change.

Reply:
Sure, but then we could split the symbols apart at that point